src/movies.js

```diff
@@ -1,25 +1,131 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+    const directors = moviesArray.map(el => el.director);
+    return directors;                      
```

Comment:
De 10! pero te meto refactor a como queremos que lo vayais poniendo vale???

```suggestion
    return moviesArray.map(el => el.director);
```